swarm/src/behaviour.rs

```diff
@@ -721,3 +848,183 @@ impl Default for CloseConnection {
         CloseConnection::All
     }
 }
+
+pub enum FromSwarm<'a, Handler: IntoConnectionHandler> {
```

Comment:
@thomaseizinger @elenaf9 as per https://github.com/libp2p/rust-libp2p/pull/2867/files#r974184169, renamed `InEvent` to `FromSwarm`